src/datasource.ts

```diff
@@ -93,16 +119,47 @@ export class DataFrameDataSource extends DataSourceApi<DataframeQuery> {
     const conversions = columns.map(({ name, dataType }) => ({
       targetField: name,
       destinationType: this.getFieldType(dataType),
+      dateFormat: 'YYYY-MM-DDTHH:mm:ss.SZ',
```

Comment:
The dates we get back from the service have fractional precision, so this is necessary to convert them correctly to unix timestamps.